README.md

```diff
@@ -130,7 +130,11 @@ Compile Sass as changes are made:
 
 ### Deployment
 
-*Likely only useful for 18F team members*
+*Likely only useful for 18F FEC team members*
+
+We use Travis for automated deploys after tests pass. If you want to deploy something it is much better to push an empty commit with a tag than doing a manual deploy.
```

Comment:
@LindsayYoung, looks like this is still outstanding and we should probably get it merged - would adding this example be okay?